service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpConnectionHandler.java

```diff
@@ -130,6 +177,15 @@ public void onConnectionBound(Event event)
         }
     }
 
+    @Override
+    public void onConnectionInit(Event event)
```

Comment:
This logic is now common for all the classes that already extend this class, so it is just abstracted up a bit